package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "metis-ui",
-  "version": "2.0-development",
+  "version": "1.7.0-development",
```

Comment:
This doesn't quite make sense to me...

Reply:
I'd created a dev branch for development with that single change (and since deleted it) - this was to restore...